packages/markdown-docx/src/ToOOXMLVisitor/rules.js

```diff
@@ -121,26 +122,28 @@ const STRONG_RULE = () => {
 /**
  * Inserts a variable.
  *
- * @param {string} title Title of the variable. Eg. receiver-1, shipper-1
- * @param {string} tag   Name of the variable. Eg. receiver, shipper
- * @param {string} value Value of the variable
- * @param {string} type  Type of the variable - Long, Double, etc.
+ * @param {string}  title  Title of the variable. Eg. receiver-1, shipper-1
+ * @param {string}  tag    Name of the variable. Eg. receiver, shipper
+ * @param {string}  value  Value of the variable
+ * @param {string}  type   Type of the variable - Long, Double, etc.
+ * @param {boolean} vanish Should vanish property be present
```

Comment:
Add a link to the `vanish` property in the doc strings.

Reply:
@algomaster99 
I have already linked it.

Reply:
I cannot see the link. I meant the link to the OOXML docs. Otherwise, you can explain that it is equivalent to `display: none`.

Reply:
@algomaster99 Oh sure. I meant you were asking for it to be present in the docstrings. I will add a link.